src/plots/plots.js

```diff
@@ -2252,14 +2252,6 @@ plots.transition = function(gd, data, layout, traces, frameOpts, transitionOpts)
         var module = contFull._module;
 
         if(!module) continue;
-
-        if(!module.animatable) {
-            var thisUpdate = {};
-
-            for(var ai in data[i]) {
-                thisUpdate[ai] = [data[i][ai]];
```

Comment:
Strange. I wonder what Ricky had in mind here. Good catch :ok_hand:

Reply:
Could've been just some initial logic that remained there.